### ISSUE TITLE
Update grommet.io to comply with WCAG tools

### ISSUE THREAD

**Issue by bryanjacquot** [grommet/grommet#685]

**Comment by melodymorgan** [comment]
Heads up... v2.grommet.io has mutliple WCAG 2.0 (A) and WCAG 2.0 (AA) Violations when inspected with axe-core.
